lightstep/resource_metric_dashboard.go

```diff
@@ -126,46 +126,37 @@ func getGroupSchema(chartSchemaType ChartSchemaType) map[string]*schema.Schema {
 				Schema: getChartSchema(chartSchemaType),
 			},
 		},
-	}
-}
-
-func getChartSchema(chartSchemaType ChartSchemaType) map[string]*schema.Schema {
-
-	var querySchema map[string]*schema.Schema
-	if chartSchemaType == UnifiedChartSchema {
-		querySchema = getUnifiedQuerySchemaMap()
-		querySchema["dependency_map_options"] = &schema.Schema{
+		"text_panel": {
 			Type:     schema.TypeList,
 			Optional: true,
-			MaxItems: 1,
+			Computed: true, // the panels can be mutated individually; chart mutations should not trigger group updates
```

Comment:
Following the pattern we used for "charts" and setting `Computed` to `true`, see https://github.com/lightstep/terraform-provider-lightstep/pull/152

Reply:
Also "text_panel" uses `TypeList` (which retains the order) rather than `TypeSet`. I'm not aware of why `TypeSet` was used for charts, but it seems only to make things slightly more complex since order is not retained.  (If I'm missing anything here, this is easy to change.)